.github/workflows/e2e-backend-switching-reuseable.yml

```diff
@@ -60,6 +58,9 @@ jobs:
           # generate cluster name and export it to Github env for cleanup step to access it.
           export CLUSTER_NAME="${CLUSTER_PREFIX}$(openssl rand -hex 2)"
           echo "CLUSTER_NAME=${CLUSTER_NAME}" >> $GITHUB_ENV
+          # set random region for AWS.
+          export GARDENER_REGION=$(./scripts/gardener/aws/get_random_region.sh)
+          export GARDENER_ZONES="${GARDENER_REGION}a"
```

Comment:
why do we always use zones with the suffix `a``

Reply:
We are always using the first zone in the region, we do not have any preference for zones.